### PR TITLE
fix(DEV-11940): fix toasts for common erorrs

### PR DESCRIPTION
**Pull request by hypsteria**
Show toast for common errors in onboarding form (not only errors in fields)

Screenshot 2024-08-08 at 10 20 53 AM

### CODE REVIEW

File: packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts [hunk 1]
```diff
@@ -77,15 +78,19 @@ export function useOnboardingPerson(): OnboardingPersonReturnType {
     isPending: isCreateLoading,
     error: createPersonError,
   } = api.persons.postPersons.useMutation(undefined, {
-    onError: () => {},
+    onError: (error) => {
+      showErrorToast(error);
```
Comment (radist2s) [inline packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts:82]: We have a special function for that: `getAPIErrorMessage(...)`.

Comment (hypsteria) [reply packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts:82]: @radist2s getAPIErrorMessage shows every message from backend. We use new function to show toasts only in cases when we have error.message. In cases when we have error.details we show errors in fields. Initially I set this function empty `onError: () => {}` to not to show any toasts, but we need this toasts for common errors

Comment (radist2s) [reply packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts:82]: Sorry, thanks, didn't realise that from the start. Can I ask you to add a comment to the function to make it clear what the exception is for?

**Review by radist2s — APPROVED** [review]

File: packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts (hunk 1 above)
Comment (hypsteria) [reply packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts:82]: @radist2s yes, thanks. Added description to showErrorToast.ts

**Pull request merged**